app/dev.js

```diff
@@ -45,7 +46,12 @@ app.on('app:window-created', async () => {
 });
 
 try {
+
+  // reload on app changes
   require('electron-reloader')(module);
```

Comment:
What is the benefit of keeping this dude around?

I imagine we can simply reload on any `app/**` changes, too, right.

Just a suggestion, of course.

Reply:
I think we could also just use `electron-reloader` alone, but that would require moving the dev script upwards. I'll push a change shortly.

Reply:
Have a look now. One tool to reload both parts.